src/engraving/dom/chord.cpp

```diff
@@ -778,6 +778,9 @@ void Chord::remove(EngravingItem* e)
     break;
 
     case ElementType::ARPEGGIO:
+        if (m_spanArpeggio == m_arpeggio) {
+            m_spanArpeggio = 0;
```

Comment:
I think it's really preferable to use `nullptr` instead of `0` in new code wherever appropriate; and perhaps even adjust existing code that still uses `0`, which is often confusing and a bit of a dated practice, as far as I know.